addons/website/static/src/snippets/s_image_gallery/gallery_slider.js

```diff
@@ -71,6 +95,17 @@ export class GallerySlider extends Interaction {
         this.onSlidCarousel();
     }
 
+    start() {
+        if (this.carouselItemEls) {
+            const firstImageEl = this.carouselItemEls.item(0).querySelector("img");
+            if (firstImageEl.complete) {
+                this.prefetchImage(1);
+            } else {
+                this.addListener(firstImageEl, "load", () => this.prefetchImage(1), { once: true });
+            }
+        }
```

Comment:
In "Image Gallery", we can replace the image by another media (like an icon or a video), in which case this prefetch fails (there is a traceback) :+1: 
=> You should add checks that there is well an image in the current slide, as well as in the target slide, before prefetching them :slightly_smiling_face: